app/controllers/reviews_controller.rb

```diff
@@ -10,6 +10,6 @@ def new
   private
 
   def review_params
-    params.require(:review).permit(:review, :rating, :book_id)
+    params.require(:review).permit(:review, :rating, :book_id, :user_id)
```

Comment:
why don't you use current_user directly here ? We don't need to explicitly get that from params